src/resources/boot.lua

```diff
@@ -109,6 +109,7 @@ function lovr.boot()
 end
 
 function lovr.run()
+  local hasWindow = lovr.graphics.hasWindow()
```

Comment:
It is possible to delay window creation until after boot time, so `lovr.graphics.hasWindow` can return different results.  For this reason I think it should not be cached and should be invoked down below on every frame.